src/file-operations.js

```diff
@@ -17,9 +30,14 @@ function readFile(file, readDirPath) {
 function parseFile(fileData, { templateName, packageName, app, rnVersion }) {
   let kotlinPackage;
   let javaPackage;
-
+  var version = rnVersion;
+  try {
+    validate(version);
+  } catch (e) {
+    version = null;
+  }
   // TODO: figure out a better way to handle one off breaking changes
-  if (rnVersion && compareVersions(rnVersion, "0.47.2") < 0) {
+  if (version && compareVersions(version, "0.47.2") < 0) {
```

Comment:
@dlowder-salesforce I am realizing that your version check just validates a github repo as a valid source, however you don't get the actual version from it. How will `compareVersions` work since I don't see support for github repos in the package.

Reply:
Actually a github repo string will fail the validation check, so `version` will be set to null in that case.

Reply:
I see...so I am actually wondering if this PR will cover this issue too and just needs a test https://github.com/peggyrayzis/react-native-create-bridge/pull/37/files

Reply:
Yeah that looks like a similar fix

Reply:
@dlowder-salesforce Since there are redundancies Im going to add add you test into the other PR and make sure it is still working as expected. I'll add you as a reviewer when it's done.

Reply:
@jarretmoses sounds good, I'll close this PR then